### PR TITLE
Refresh token added

### PR DESCRIPTION
**Pull request by Nikita5796**

### CODE REVIEW

File: Controllers/UsersController.cs [hunk 1]
```diff
@@ -21,7 +22,7 @@ public class UsersController : ControllerBase
 
         private IUserService _userService;
         JWTHelper jwtHelper = new JWTHelper();
-        //RefreshTokenHelper refToken = new RefreshTokenHelper();
+        RefreshTokenHelper refToken = new RefreshTokenHelper();
```
Comment (ketannaik) [inline Controllers/UsersController.cs]: Move this line inside the "if condition" on line 40

File: Helpers/RefreshTokenHelper.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+
+namespace BookAppStoreAPI.Helpers
+{
+    public class RefreshTokenHelper
+    {
+        public RefreshToken Generate_Refresh_Token()
+        {
+            RefreshToken refToken = new RefreshToken();
```
Comment (ketannaik) [inline Helpers/RefreshTokenHelper.cs:14]: Where is Refresh Token sent as the return type here? Also, where is the RefreshToken model?

**Review by ketannaik — CHANGES_REQUESTED** [review]

**Review by bala0105 — COMMENTED** [review]
Any specific reason for committing .dll and .pdb files?

File: Controllers/UsersController.cs (hunk 1 above)
Comment (Nikita5796) [reply Controllers/UsersController.cs]: done.

File: Helpers/RefreshTokenHelper.cs (hunk 2 above)
Comment (Nikita5796) [reply Helpers/RefreshTokenHelper.cs:14]: RefreshToken.cs file is entity class created right inside Entities folder. using object of refreshtoken.cs I am storing it into refresh token table.

**Review by ketannaik — APPROVED** [review]

**Review by bala0105 — APPROVED** [review]

**Pull request merged**